### ISSUE TITLE
Make 'modelVersion' optional

### ISSUE THREAD

**Issue by miere** [takari/polyglot-maven#180]
For every single maven project I've been created I had to define `<modelVersion>4.0.0</modelVersion>`. Luckily, Maven Polyglot YAML makes it less verbose/annoying but still an inconvenient in which sometimes, when I miss to define it, I just got my project build failing.

What about making it optional and make it assume `4.0.0` as default value in case of being omitted?

**Comment by mosabua** [comment]
To be honest we want to avoid magic things like that but rather have a straight conversion.

**Comment by mosabua** [comment]
If at all, it would have to be consistent and the same in all dialects and work cleanly for import and export back and forth. Also we would then, in the future when new model version is used, figure out changes of default value or whatever.

**Comment by lefou** [comment]
In fact, we already have this default in the scala dialect: f765ca461917b101a04d3b4776afe837efe34bb3.

**Comment by Tradunsky** [comment]
All boilerplate stuff are optional in polyglot-kotlin as well as model version.

**Comment by mosabua** [comment]
Fair enough .. then please send a PR that does a similar thing for the YAML and ideally all other dialects.
